tests/unit/AddressListSm.spec.ts

```diff
@@ -259,4 +259,64 @@ describe('AddressListSm', () => {
       done()
     })
   })
+
+  it('displays complete your filing to display', async () => {
+    store.state.entityType = 'BC'
+    const wrapper = mount(AddressListSm,
+      {
+        store,
+        vuetify,
+        propsData: {
+          completedFilingRequired: true
+        }
+      })
+    const expectedMessage = 'Complete your filing to display'
+    const vm = wrapper.vm as any
+    // Click the records office tab to display the addresses
+    const button = vm.$el.querySelector('#records-office-panel-toggle')
+    await button.click()
+    await Vue.nextTick()
+
+    expect(vm.$el.querySelector('#registered-office-panel .delivery-address-list-item .complete-filing')
+      .textContent).toBe(expectedMessage)
+
+    expect(vm.$el.querySelector('#registered-office-panel .mailing-address-list-item .complete-filing')
+      .textContent).toBe(expectedMessage)
+
+    expect(vm.$el.querySelector('#records-office-panel .delivery-address-list-item .complete-filing')
+      .textContent).toBe(expectedMessage)
+
+    expect(vm.$el.querySelector('#records-office-panel .mailing-address-list-item .complete-filing')
```

Comment:
Good tests for BCOMP 👍 

If you're in this file again, I think we should move as much as possible to `wrapper.find()` to select elements instead of using `vm.$el.querySelector()`. It's cleaner and has richer methods :)

Reply:
Fixed, thanks!